audio/Makefile

```diff
@@ -1,55 +1,23 @@
-PHONY := all package clean
-rwildcard=$(foreach d,$(wildcard $1*),$(call rwildcard,$d/,$2) $(filter $(subst *,%,$2),$d))
-
-CC := arm-vita-eabi-gcc
-CXX := arm-vita-eabi-g++
-STRIP := arm-vita-eabi-strip
-
-PROJECT_TITLE := VitaSDK sample
-PROJECT_TITLEID := VSDK00001
-
-PROJECT := vita_sample
-CXXFLAGS += -std=c++11 -I../common
-CFLAGS += -std=gnu11 -I../common
-
-LIBS := -lSceDisplay_stub -lSceCtrl_stub  -lSceAudio_stub 
-
-SRC_C :=$(call rwildcard, src/, *.c)
-SRC_CPP :=$(call rwildcard, src/, *.cpp)
-
-OBJ_DIRS := $(addprefix out/, $(dir $(SRC_C:src/%.c=%.o))) $(addprefix out/, $(dir $(SRC_CPP:src/%.cpp=%.o)))
-OBJS := $(addprefix out/, $(SRC_C:src/%.c=%.o)) $(addprefix out/, $(SRC_CPP:src/%.cpp=%.o))
-
+CFLAGS += -I../common
+STUBS := Display Ctrl Audio
+LIBS := -lm
+#no need to modify the following
 
+PROJECT := $(shell basename $(CURDIR))
+PHONY := all package clean
 all: package
-
 package: $(PROJECT).vpk
-
-$(PROJECT).vpk: eboot.bin param.sfo
-	vita-pack-vpk -s param.sfo -b eboot.bin $(PROJECT).vpk
-	
-eboot.bin: $(PROJECT).velf
-	vita-make-fself $(PROJECT).velf eboot.bin
-
-param.sfo:
-	vita-mksfoex -s TITLE_ID="$(PROJECT_TITLEID)" "$(PROJECT_TITLE)" param.sfo
-
+clean:
+	rm -rf $(PROJECT).*
+
+$(PROJECT).vpk: $(PROJECT).sfo $(PROJECT).fself
+	vita-pack-vpk -s $< -b $(word 2,$^) $(foreach A,$(wildcard sce_sys/*.*) $(wildcard sce_sys/*/*/*.*),--add $A=$A) $@
+$(PROJECT).fself: $(PROJECT).velf
+	vita-make-fself $< $@
```

Comment:
"-c" will create a compressed SELF, but I don't want there to be a "VITA_MAKE_FSELF_FLAGS" because then it requires developers to know how to use `vita-make-fself`. A proper way would be to have some variable like `VITA_COMPRESSED` set (also since we encourage compressed as default, it should be `VITA_UNCOMPRESSED`). Also, "-s" creates safe homebrew so we also need "VITA_SAFE_HOMEBREW" or something. Of course parsing all this shouldn't be in every makefile.